cmd/headscale/cli/init.go

```diff
@@ -0,0 +1,75 @@
+package cli
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/juanfont/headscale"
+	"github.com/spf13/cobra"
+	"tailscale.com/types/wgkey"
+)
+
+var InitCmd = &cobra.Command{
+	Use:   "init",
+	Short: "Creates a basic Headscale env",
+}
+
+var InitSqliteCmd = &cobra.Command{
+	Use:   "sqlite",
+	Short: "Creates a headscale env using SQLite as database",
+	Run: func(cmd *cobra.Command, args []string) {
+		force, _ := cmd.Flags().GetBool("force")
+		if !force {
+			if _, err := os.Stat("config.json"); err == nil {
+				fmt.Println("config.json already exists")
+				return
+			}
+			if _, err := os.Stat("private.key"); err == nil {
+				fmt.Println("private.key already exists")
+				return
+			}
+			if _, err := os.Stat("derp.yaml"); err == nil {
+				fmt.Println("derp.yaml already exists")
+				return
+			}
+		}
+
+		fmt.Println("Creating config.json")
+		cfg := `{
+		"server_url": "http://127.0.0.1:8000",
+		"listen_addr": "0.0.0.0:8000",
```

Comment:
Why by default listen on *? I rather would prefer a more strict default, like 127.0.0.1. Even thou it is also the default in the config examples.